src/main/java/org/jabref/gui/linkedfile/LinkedFileDialogController.java

```diff
@@ -27,26 +30,46 @@ public class LinkedFileEditDialogView extends BaseDialog<LinkedFile> {
 
     @Inject private DialogService dialogService;
     @Inject private StateManager stateManager;
-
     @Inject private PreferencesService preferences;
 
     private LinkedFilesEditDialogViewModel viewModel;
-
     private final LinkedFile linkedFile;
 
-    public LinkedFileEditDialogView(LinkedFile linkedFile) {
+    /**
+     * Constructor for adding a new LinkedFile.
+     */
+    public LinkedFileDialogController() {
+        this(new LinkedFile("", "", ""));
+        initializeDialog(Localization.lang("Add file link"), ADD_BUTTON);
```

Comment:
Missing localization key. Refer to point 4 of the coding FAQs.
https://devdocs.jabref.org/code-howtos/faq.html

Reply:
I checked `JabRef_en.properties` - and following entry exists:

```properties
Add\ file\ link=Add file link
```

Normally `org.jabref.logic.l10n.LocalizationConsistencyTest#findMissingLocalizationKeys`, should find all missing localization keys.

Reply:
Yes, it was added in the commit after this review.

Reply:
Yes, it was added on the commit after this review.